risc0/zkvm/src/receipt/succinct.rs

```diff
@@ -35,16 +35,17 @@ use risc0_zkp::{
         hash::{hash_suite_from_name, sha::Sha256},
     },
     verify::VerificationError,
+    MAX_CYCLES_PO2,
```

Comment:
This will need to be selective based on the unstable feature, which is why it avoided being brought in with a `using`.